src/middlewared/middlewared/etc_files/pykmip/pykmip.conf

```diff
@@ -0,0 +1,14 @@
+<%
+	kmip_config = middleware.call_sync('kmip.config')
+	cert = middleware.call_sync('certificate.query', [['id', '=', kmip_config['certificate']]])
+	ca = middleware.call_sync('certificateauthority.query', [['id', '=', kmip_config['certificate_authority']]])
```

Comment:
Seems we should have `get: True`

Reply:
We don't as it will raise exception if `kmip_config['certificate']` is `None`

Reply:
Well, we dont check if cert or ca are empty, so it will raise an IndexError wither way

Reply:
Actually, we do, I am blind, again :D